tests/integration/defs/test_e2e.py

```
@@ -2012,23 +1860,16 @@ def test_ptp_quickstart_advanced(llm_root, llm_venv, model_name, model_path):
-            if "Llama3.1-70B" in model_name or "Llama3.3-70B" in model_name:
-                cmds.append(f"--max_num_tokens=1024")
-            llm_venv.run_cmd(cmds, stdout=running_log)
-            if model_name in mapping:
-                _check_mem_usage(running_log, [mapping[model_name], 0, 0, 0])
+        cmds = [
+            str(example_root / "quickstart_advanced.py"),
+            "--enable_chunked_prefill",
+            f"--model_dir={llm_models_root()}/{model_path}",
+        ]
+        if "Qwen3" in model_name:
+            cmds.append(f"--kv_cache_fraction=0.6")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)